src/swap/SwapScreen.test.tsx

```diff
@@ -142,27 +146,106 @@ describe('SwapScreen', () => {
   })
 
   it('should keep the to amount in sync with the exchange rate', () => {
-    const { swapFromContainer, swapToContainer, getByText } = renderScreen()
+    mockFetch.mockResponse(
+      JSON.stringify({
+        unvalidatedSwapTransaction: {
+          price: '1.2345678',
+        },
+      })
+    )
+    const { swapFromContainer, swapToContainer, getByText } = renderScreen({})
 
     void act(() => {
       fireEvent.changeText(within(swapFromContainer).getByTestId('SwapAmountInput/Input'), '1.234')
       jest.runAllTimers()
     })
 
+    expect(getByText('1 CELO ≈ 1.23456 cUSD')).toBeTruthy()
     expect(within(swapFromContainer).getByTestId('SwapAmountInput/Input').props.value).toBe('1.234')
-    expect(within(swapToContainer).getByTestId('SwapAmountInput/Input').props.value).toBe('4.10305')
+    expect(within(swapToContainer).getByTestId('SwapAmountInput/Input').props.value).toBe(
+      '1.5234566652'
+    )
     expect(getByText('swapScreen.review')).not.toBeDisabled()
   })
 
   it('should keep the from amount in sync with the exchange rate', () => {
-    const { swapFromContainer, swapToContainer } = renderScreen()
+    mockFetch.mockResponse(
+      JSON.stringify({
+        unvalidatedSwapTransaction: {
+          price: '0.12345678',
+        },
+      })
+    )
+    const { swapFromContainer, swapToContainer, getByText } = renderScreen({})
+
+    void act(() => {
+      fireEvent.changeText(within(swapToContainer).getByTestId('SwapAmountInput/Input'), '1.234')
+      jest.runAllTimers()
+    })
+
+    expect(getByText('1 CELO ≈ 8.10000 cUSD')).toBeTruthy()
+    expect(within(swapFromContainer).getByTestId('SwapAmountInput/Input').props.value).toBe(
```

Comment:
Shouldn't this be `SwapAmountIOutput/Output` ??
I seeing the test is passing, I'm wondering if this test even work. 😅